lib/friendly_shipping/rest_client.rb

```diff
@@ -32,6 +32,22 @@ def post(path, payload, headers)
           Failure(error)
         end
       end
+
+      def put(path, payload, headers)
+        Success(
+          ::RestClient.put(
+            path,
+            payload,
+            headers
+          )
+        )
+      rescue ::RestClient::ExceptionWithResponse => error
+        if error.to_s == '400 Bad Request'
```

Comment:
`RestClient::ExceptionWithResponse` also implements `#http_code`. So instead of casting a String and comparing it to another (that could change) we could just match integers here. Nothing that holdsback this PR, but maybe worth refactoring in the other methods as well?

Reply:
Thanks, I'm including a commit that fixes this. When reading `RestClient`s `exceptions.rb` I also realized I'm rescueing a compatibility layer class, so fixing that as well.